requirements-dev.txt

```diff
@@ -3,7 +3,8 @@
 codecov
 coverage
 flake8
-ophyd ==1.3.4rc1
+# TEMPORARY UNTIL https://github.com/NSLS-II/ophyd/pull/682 IS RELEASED
```

Comment:
This was a pretty bad mistake on my part. I meant to just try 1.3.4rc1 in the pull request.

Reply:
For some reason ophyd-for-suitcase-utils is causing an error because curio is not available and caproto wants it, while ophyd 1.3.4rc1 works.

Reply:
Is it possible we could move away from ophyd-for-suitcase-utils? I can't remember the other related issues....